rest/netutil.go

```diff
@@ -130,11 +130,15 @@ func (c *Client) RestAPICall(method Method, path string, options interface{}) ([
 			return nil, err
 		}
 		log.Debugf("*** options => %+v", bytes.NewBuffer(OptionsJSON))
+		ioutil.WriteFile("jsons.json", OptionsJSON, 0644)
 		req, err = http.NewRequest(method.String(), reqUrl.String(), bytes.NewBuffer(OptionsJSON))
 	} else {
 		req, err = http.NewRequest(method.String(), reqUrl.String(), nil)
 	}
 
+	file, _ := json.MarshalIndent(options, "", " ")
+	ioutil.WriteFile("2inut.json", file, 0644)
```

Comment:
same as above

Reply:
It was for debugging purpose. Now I have removed.